product_docs/docs/biganimal/release/overview/03_security/compliance_and_certifications.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Security compliance and certifications
+navTitle: Compliance and certifications
+---
+
+BigAnimal adheres to the following security standards, certifications, and capabilities: 
```

Comment:
I don't think we need to include the SSO and RBAC capabilities sections. They are covered, albeit in much less detail in terms of conceptual information, in the parent topic and the Managing access topic. @amrita-suresh , what are your thoughts? Now that you see the content in the topic, do you think it is worth keeping it? I'm on the fence. I don't think it does any harm.

Reply:
I don't think we need to include in-depth sections on it either. I would defer to Aislinn & Ben though.